admin/access.php

```diff
@@ -90,7 +90,7 @@
         $access = new Access(filter_input(INPUT_GET, 'access_id', FILTER_SANITIZE_SPECIAL_CHARS));
         $access->update($_POST);
         if (!AmpError::occurred()) {
-            show_confirmation(T_('Updated'), T_('Access List Entry updated'), AmpConfig::get('web_path') . '/admin/access.php');
+            show_confirmation(T_('Updated'), T_('Access List entry updated'), AmpConfig::get('web_path') . '/admin/access.php');
```

Comment:
Why one time a long version with Ampache personally speaking to the user and one time not?

I'd either do one or the other like:

Your Access List entry has been removed
and
Your Access List entry has been updated

Or

Access List entry removed
and
Access List entry updated

The all in all question is: Should Ampache speak personal/natural to the user or more generic? ^^

Reply:
@kuzi-moto this is probably the last major thing we need to check before finalising this pull.
personal sentences to the user. especially where we are personal in one and impersonal in the other